src/Integrations/dashboardIntegration.ts

```diff
@@ -69,7 +69,7 @@ export async function addDataTrailPanelAction(
 
   if (subMenu.length > 0) {
     items.push({
-      text: 'Explore metrics',
+      text: 'Metrics Drilldown',
```

Comment:
I suggest keeping the casing the same:

```suggestion
      text: 'Metrics drilldown',
```